packages/syft/src/syft/service/project/project.py

```diff
@@ -657,6 +657,14 @@ def add_code_request_to_project(
 class Project(SyftObject):
     __canonical_name__ = "Project"
     __version__ = SYFT_OBJECT_VERSION_1
+    __attr_custom_repr__ = [
+        "id",
+        "name",
+        "description",
+        "user_email_address",
+        "events",
+        "user_signing_key",
```

Comment:
Could we remove signing key from repr, as it is a sensitive credential, we could avoid printing it?

Reply:
Sorry, it is my mistake, I looked it in ticket, I found that, I incorrectly mentioned signing key.

Reply:
Resolved in [4c7f7db](https://github.com/OpenMined/PySyft/pull/7717/commits/4c7f7dba1511442c166a9796122bdb4ab13c0ff0)